ansible_collections/arista/cvp/changelogs/fragments_backup/v3.10.0.yml

```diff
@@ -0,0 +1,2 @@
+release_summary: |
+  Release 3.10.0 - See documentation on cvp.avd.sh for details.
```

Comment:
we should add to bump in requirements to the changelog. Categorize under `minor_changes:`